fact_store/model.py

```diff
@@ -0,0 +1,56 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String, Float, Boolean, Sequence
+
+
+Base = declarative_base()
+
+
+class EventModel(Base):
+    """
+    Model Used for persisting events when model generates a prediction.
+    We track this for quality control
+    """
+    __tablename__ = "events"
+    id = Column(Integer, primary_key=True, autoincrement=True)
```

Comment:
Why use separate `id` if `predict_id` is unique anyway?

Reply:
We can make the predict_id unique and yes we can make it the primary key

Reply:
Fixed.